testsupport/tiers/checks.go

```diff
@@ -1153,6 +1154,14 @@ func gitOpsServiceLabel() namespaceObjectsCheck {
 	}
 }
 
+func environment(name string) namespaceObjectsCheck {
+	return func(t *testing.T, ns *corev1.Namespace, memberAwait *wait.MemberAwaitility, _ string) {
+		// TODO fix for migration/existing namespaces cases
```

Comment:
>// TODO fix for migration/existing namespaces cases

what do you mean by that?

Reply:
never mind, it's a remainder of a quick copy/paste of the [gitOpsServiceLabel func](https://github.com/codeready-toolchain/toolchain-e2e/blob/master/testsupport/tiers/checks.go#L1146)

Reply:
removed in 473dad2